UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1155,11 +1155,17 @@ public string GenerateMD5(string filename)
         private async Task LoadGMLCache(string filename, LoaderDialog dialog = null)
         {
             await Task.Run(() => {
-                if (SettingsWindow.UseGMLCache && File.Exists(Path.Join("GMLCache", "index")))
+                if (SettingsWindow.UseGMLCache)
```

Comment:
Probably better for future PR, but this (and savegmlcache) if condition can be negated.